### PR TITLE
Issue #1699: Notification badge should be round.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/1699

### CODE REVIEW

File: core/modules/admin_bar/css/admin_bar.css [hunk 1]
```diff
@@ -313,15 +313,24 @@ tr.admin-bar-tweak-permissions-processed td.module {
...
-  border-radius: 50%;
   text-align: center;
+  width: 22px;
+  height: 22px;
+  line-height: 20px;
   background-color: #ee3d23;
+  border-radius: 50%;
+  position: absolute;
+  top: 8px;
+  left: 12t bNotification badge should be roundpx;
```
Comment (klonos) [inline core/modules/admin_bar/css/admin_bar.css:329]: ! accidental paste there?

**Comment by klonos** [comment]
The logic of having something square (same width & height instead of just min width) with 50% radius rounded corners seems right to me in theory, but do we have a way to actually trigger the badge in order to actually preview the thing and test across different browsers?

**Comment by jenlampton** [comment]
To test change the files directory to something non-writable like /core/
under config . Media . File system.
- -
  Please excuse brevity and typos. Message typed with my thumbs from an Open
  Source OS.

**Comment by quicksketch** [comment]
Geoff took another stab at this here: https://github.com/backdrop/backdrop/pull/1514.

**Pull request closed without merging**